CMake/telesculptor-external-kwiver.cmake

```diff
@@ -77,6 +78,7 @@ ExternalProject_Add(kwiver
     -DADDITIONAL_C_FLAGS:STRING=${ADDITIONAL_C_FLAGS}
     -DADDITIONAL_CXX_FLAGS:STRING=${ADDITIONAL_CXX_FLAGS}
   INSTALL_COMMAND ${CMAKE_COMMAND} -E echo "Skipping install step."
+  USES_TERMINAL_BUILD 1
```

Comment:
I'm curious about what this does.

Reply:
From 890ecc70aee48d317b3478bee5acc23cd63a6bb9:
> Use terminal to build externals
>
> Enable direct use of terminal for the build steps of fletch and KWIVER. This mostly affects Ninja, forcing these builds to be serialized but also allowing the output to be seen during the build.